enderio-base/src/main/java/com/enderio/base/common/item/EIOItems.java

```diff
@@ -371,6 +384,65 @@
 
     // endregion
 
+
+    // region DarkSteel
+
+    public static final Tag.Named<Block> DARK_STEEL_TIER_TAG = BlockTags.createOptional(EnderIO.loc("needs_dark_steel"));
+
+    public static final Tier DARK_STEEL_TIER = TierSortingRegistry.registerTier(
+        new ForgeTier(3, 2000, 8.0F, 3, 25, DARK_STEEL_TIER_TAG, () -> Ingredient.of(EIOItems.DARK_STEEL_INGOT.get())),
+        EnderIO.loc("dark_steel_tier"), List.of(Tiers.DIAMOND),
+        List.of(Tiers.NETHERITE));
+
+    public static ItemEntry<DarkSteelPickaxeItem> DARK_STEEL_PICKAXE = REGISTRATE
+        .item("dark_steel_pickaxe", DarkSteelPickaxeItem::new)
+        .group(new NonNullLazyValue<>(() -> EIOCreativeTabs.GEAR))
+        .onRegister(item -> DarkSteelUpgradeRegistry
+            .instance()
+            .addUpgradesForItem(Objects.requireNonNull(item.getRegistryName()), EmpoweredUpgrade.NAME, SpoonUpgrade.NAME))
+        .register();
+
+    public static ItemEntry<DarkSteelAxeItem> DARK_STEEL_AXE = REGISTRATE
+        .item("dark_steel_axe", DarkSteelAxeItem::new)
+        .group(new NonNullLazyValue<>(() -> EIOCreativeTabs.GEAR))
+        .onRegister(item -> DarkSteelUpgradeRegistry
+            .instance()
+            .addUpgradesForItem(Objects.requireNonNull(item.getRegistryName()), EmpoweredUpgrade.NAME, ForkUpgrade.NAME))
+        .register();
+
+    //TODO: use Config for required levels
+    public static ItemEntry<DarkSteelUpgradeItem> DARK_STEEL_UPGRADE_EMPOWERED_1 = REGISTRATE
+        .item("dark_steel_upgrade_empowered_1", properties -> new DarkSteelUpgradeItem(properties,4, EmpoweredUpgrade.TIER_0))
```

Comment:
```suggestion
        .item("dark_steel_upgrade_empowered_1", properties -> new DarkSteelUpgradeItem(properties, 4, EmpoweredUpgrade.TIER_0))
```
add space here and on the other itemEntries as well